fvm/fvm_signature_test.go

```diff
@@ -122,11 +122,13 @@ func TestKeyListSignature(t *testing.T) {
                           return keyList.verify(
                               signatureSet: signatureSet,
                               signedData: message,
+							  domainSeparationTag: "%s"
```

Comment:
maybe convert these tabs to spaces:
```suggestion
                              signedData: message,
                              domainSeparationTag: "%s"
```